mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/EmptyStacktraceScenario.kt

```diff
@@ -15,6 +16,7 @@ internal class EmptyStacktraceScenario(config: Configuration,
         Bugsnag.notify(EmptyException("EmptyStacktraceScenario"))
     }
 
+    @SuppressLint("NewApi")
```

Comment:
AGP uses a newer version of android lint which detects this Throwable constructor is only available from API 19 onwards, however, we don't currently run mazerunner on that API level.